keyboards/giabalanaipico/giabalanaipico.c

```diff
@@ -0,0 +1,226 @@
+
+        { 0,      1,      2,   3,   4,   5,   6  },
+        { 23,     22,     21,  20,  19,  18,  17 },
+        { 24,     25,     26,  27,  28,  29,  30 },
+        { 47,     46,     45,  44,  43,  42,  41 },
+        { 48,     49,     50,  51,  52,  53,  54 },
+
+
+        { 60,     61,     62,  63,  64,  65,  66  },
+        { 84,     83,     82,  81,  80,  79,  78  },
+        { 85,     86,     87,  88,  89,  90,  91  },
+        { 110,    109,    108, 107, 106, 105, 104 },
+        { NO_LED, 111,    112, 113, 114, 115, 116 },
+
+        { NO_LED, NO_LED, 71,  70,  69,  68,  67  },
+        { NO_LED, 72,     73,  74,  75,  76,  77  },
+        { NO_LED, 97,     96,  95,  94,  93,  92  },
+        { NO_LED, 98,     99,  100, 101, 102, 103 },
+        { NO_LED, 122,    121, 120, 119, 118, 117 }
+    }, {
+        {  88,  0 }, {  80,  0 }, {  72,  0 }, {  64,  0 }, {  56,  0 }, {  48,  0 }, {  40,  0 }, {  32,  0 }, {  24,  0 }, {  16,  0 }, {   8,  0 }, {   0,  0 },
+        {   4, 16 }, {  12, 16 }, {  20, 16 }, {  28, 16 }, {  36, 16 }, {  44, 16 }, {  52, 16 }, {  60, 16 }, {  68, 16 }, {  76, 16 }, {  84, 16 }, {  92, 16 },
+        {  96, 32 }, {  88, 32 }, {  80, 32 }, {  72, 32 }, {  64, 32 }, {  56, 32 }, {  48, 32 }, {  40, 32 }, {  32, 32 }, {  24, 32 }, {  16, 32 }, {   8, 32 },
+        {  12, 48 }, {  20, 48 }, {  28, 48 }, {  36, 48 }, {  44, 48 }, {  52, 48 }, {  60, 48 }, {  68, 48 }, {  76, 48 }, {  84, 48 }, {  92, 48 }, { 100, 48 },
+        { 104, 64 }, {  96, 64 }, {  88, 64 }, {  80, 64 }, {  72, 64 }, {  64, 64 }, {  56, 64 }, {  48, 64 }, {  40, 64 }, {  32, 64 }, {  24, 64 }, {  16, 64 },
+
+        { 120,  0 }, { 128,  0 }, { 136,  0 }, { 144,  0 }, { 152,  0 }, { 160,  0 }, { 168,  0 }, { 176,  0 }, { 184,  0 }, { 192,  0 }, { 200,  0 }, { 208,  0 },
+        { 212, 16 }, { 204, 16 }, { 196, 16 }, { 188, 16 }, { 180, 16 }, { 172, 16 }, { 164, 16 }, { 156, 16 }, { 148, 16 }, { 140, 16 }, { 132, 16 }, { 124, 16 }, { 116, 16 },
+        { 120, 32 }, { 128, 32 }, { 136, 32 }, { 144, 32 }, { 152, 32 }, { 160, 32 }, { 168, 32 }, { 176, 32 }, { 184, 32 }, { 192, 32 }, { 200, 32 }, { 208, 32 },        { 224, 32 },
+        { 212, 48 }, { 204, 48 }, { 196, 48 }, { 188, 48 }, { 180, 48 }, { 172, 48 }, { 164, 48 }, { 156, 48 }, { 148, 48 }, { 140, 48 }, { 132, 48 }, { 124, 48 }, { 116, 48 },
+        { 120, 64 }, { 128, 64 }, { 136, 64 }, { 144, 64 }, { 152, 64 }, { 160, 64 }, { 168, 64 }, { 176, 64 }, { 184, 64 }, { 192, 64 }, { 200, 64 }, { 208, 64 }
+    }, {
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,   4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,
+        4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4
+    }
```

Comment:
please migrate this to `rgb_matrix.layout` in `info.json`
```suggestion
```

Reply:
This was time comsuming.
I made a script to streamline the process.

https://zenn-dev.translate.goog/3araht/scraps/5532f5e565882c?_x_tr_sl=ja&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp&_x_tr_hist=true

Reply:
`qmk migrate` can do this for you.

Reply:
Are you sure?
As of Sep 4, 2023, with qmk 0.21.6, somehow it didn't work with the command below.
`qmk migrate -kb bandominedoni`
Maybe it will be supported in the future?

Reply:
Hm, maybe I was mistaken. In any case `qmk info -f json` will work too, just copy the layout into the info.json.

Reply:
`qmk info -f json` is cool. It is easier to use.
However, the result was the same. rgb_matrix-related settings were not migrated.
Seems like my script will help people for a while.

Reply:
As I said, you need to copy the output of that command. It doesn't modify any files.

Reply:
I mean, there is no output generated that is related to the RGB matrix layout.
I tested with 0.22.2 but it was the same.